scripts/build.py

```diff
@@ -278,14 +278,13 @@ def decide_what_to_build(recipes_path, python, packages, numpy):
             continue
         logger.debug('Evaluating recipe: {}'.format(recipe_dir))
 
-	    # we are essentially looping over metadata here
-        # TODO: build, run, test are redundant, remove them in the future?
+	# we are essentially looping over metadata here
         dependencies = get_deps_and_metadata(recipe_dir, python, numpy)
-        for (build, run, test, meta) in dependencies:
+        for meta in dependencies:
             try:
                 # We want to get a single path here
                 # build_cmd is no longer used in this implementation
```

Comment:
This comment can be removed to avoid confusion.

Reply:
done